src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -1432,9 +1442,54 @@
 								marqueeWidget.reset();
 							}
 						}
+
 					}
 				}
-			};
+			}
+
+			prototype._enableSwipe = function (element) {
+				ns.event.enableGesture(
```

Comment:
You enabled `drag` gesture but you never disabled it.

Reply:
Yes, you're right. Drag events are enabled once element is selected. We should disable them once element is unselected. The problem is that the element is unselected immediately after touch/mouse move before drag is started.

According to guide the selection should not disappear during swipe and there is a separate issue for that https://github.com/Samsung/TAU/issues/1227